config-linux.md

```diff
@@ -361,6 +361,7 @@ The following parameters can be specified to set up the controller:
 * **`realtimePeriod`** *(uint64, OPTIONAL)* - same as **`period`** but applies to realtime scheduler only
 * **`cpus`** *(string, OPTIONAL)* - list of CPUs the container will run in
 * **`mems`** *(string, OPTIONAL)* - list of Memory Nodes the container will run in
+* **`idle`** *(int64, OPTIONAL)* - cgroups are configured with minimum weight, 0: default behavior, 1: SCHED_IDLE.
```

Comment:
I'm getting mixed signals on the actual "type" of this value from https://github.com/torvalds/linux/commit/304000390f88d049c85e9a0958ac5567f38816ee#diff-1b49efb1a709264f46f4ac518b965e9620acc2f5b928209547313b8dfc9f1286R400 (`int`) vs https://github.com/torvalds/linux/commit/304000390f88d049c85e9a0958ac5567f38816ee#diff-1b49efb1a709264f46f4ac518b965e9620acc2f5b928209547313b8dfc9f1286R511 (`long`) -- it doesn't make a _huge_ difference since the only values are 0 and 1, but I think any future changes/improvements will benefit if we match the kernel's type size from the get go. :sweat_smile:

Any ideas which one is "canonical" ? (unfortunately I'm not very familiar with the kernel source to be able to parse this diff better than the two conflicting lines I found :see_no_evil:)

Reply:
Doest it support magic values like `-1` ? (wondering if it would make it easier to use to use an uintXX if it only supports 0 and 1

Reply:
The text around it implies that it might someday?  It does _not_ currently.  Currently it supports 0 and 1.  What I'd like to do is make sure the type we choose matches the type the kernel uses, but I guess `int64` fits both of C's `int` and `long` on all platforms I'm aware of, so I guess it should be plenty. :+1: